packages/ui/src/styles/global.css

```diff
@@ -2,34 +2,27 @@
 @tailwind components;
 @tailwind utilities;
 
-.tool-carousel{
```

Comment:
@Shiyamal-Kumar  told he made some changes to this styles after discussion with you. Hope you didn't made any changes that would effect the other carousals.